README.rst

```diff
@@ -214,7 +214,7 @@ These should work for OS X:
     $ export CONAN_COMPILER=clang
     $ export CONAN_COMPILER_VERSION=3.5
 
-Finally, there are some tests that use conan to package Go-lang
+Finally, some tests use conan to package Go-lang
```

Comment:
We have recently removed the go-lang tests, this section about go-lang might be removed.